docs/providers/documentation/bash-provider.mdx

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Also bash provider has a limitation of cloud execution, it is part of the limited providers.

Reply:
Yep, will add this